src/components/Tooltip/index.test.js

```diff
@@ -1,6 +1,7 @@
 import {mount} from "enzyme"
 import React from "react"
 import Tooltip from "./index"
+import "jest-styled-components"
```

Comment:
Makes the snapshot diffs better when using styled components. https://github.com/styled-components/jest-styled-components